File: easy/021. They see me Palindroming/Same.java
import java.util.*;

class Same {
	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		String name = input.nextLine();
		String reversed_name = new StringBuilder(name).reverse().toString();
		System.out.println(reversed_name.equals(name) ? "Yes" : "No");
	}
}

File: easy/021. They see me Palindroming/same.c
#include <stdio.h>

int main()
{
    int size = 5;
	char name[size+1];
	scanf("%s", &name);
	char reversed_name[size+1];

	for (int i = 0; i < size; i++) {
		reversed_name[i] = name[size-1 - i];
	}

	strncmp(name, reversed_name, size) == 0 ? printf("Yes") : printf("No");
	return 0;
}